packages/turf-isobands/index.js

```diff
@@ -0,0 +1,269 @@
+// https://github.com/RaumZeit/MarchingSquares.js
+var MarchingSquaresJS = require('./marchingsquares-isobands.min');
+
+var turfHelpers = require('@turf/helpers');
+var turfFeaturecollection = turfHelpers.featureCollection;
+var turfPolygon = turfHelpers.polygon;
+var turfMultiPolygon = turfHelpers.multiPolygon;
+var turfExplode = require('@turf/explode');
+var turfInside = require('@turf/inside');
+var turfArea = require('@turf/area');
+
+/*******************************************************************
+ * Takes a grid ({@link FeatureCollection}) of {@link Point} features with z-values and an array of
+ * value breaks and generates filled contour isobands.
+ *
+ *
+ * @module turf/isobands
+ * @category interpolation
```

Comment:
`@module` & `@category` is optional, this would be defined in the `documentation.yml` in `turf-www`.

https://github.com/Turfjs/turf-www/blob/gh-pages/documentation.yml

However `@name` is required = "isobands"